cirq-superstaq/cirq_superstaq/ops/qubit_gates.py

```diff
@@ -594,14 +607,15 @@ def parallel_gates_operation(*ops: cirq.Operation) -> cirq.Operation:
 
 
 class RGate(cirq.PhasedXPowGate):
-    """A single-qubit gate that rotates about an axis in the X-Y plane."""
+    """A single-qubit gate that rotates about an axis in the `X`-`Y` plane."""
 
     def __init__(self, theta: cirq.TParamVal, phi: cirq.TParamVal) -> None:
-        """Args:
-        phi: Angle (in radians) defining the axis of rotation in the `X`-`Y` plane:
-             `cos(phi) X + sin(phi) Y` (i.e. `phi` radians from `X` to `Y`).
+        """Initializes an RGate.
 
-        theta: Angle (in radians) by which to rotate.
+        Args:
+            theta: The angle (in radians) by which to rotate.
+            phi: The angle (in radians) defining the axis of rotation in the `X`-`Y` plane:
+             `cos(phi) X + sin(phi) Y` (i.e. `phi` radians from `X` to `Y`).
```

Comment:
should we be using `:math:` here? (or not using it elsewhere?) (not sure what the our protocol is for this, tbh i don't think i've ever put `:math:` in a docstring)

Reply:
I've used `:math:` in `qiskit_superstaq/custom_gates.py` for example, mainly for things like `:math: \pi` and matrices. I didn't use it here since `phi` was the name of one of the input args (unlike, say, \pi). Thought about using `:math:` for cosine, sine, X, and Y if rendering in docs was a nit.

Reply:
Is there anything wrong with copy-pasting the unicode π? My gut reaction is "yes", but I don't know.

If not, I'd argue for that. Otherwise, using `:math:` makes sense to me -- I would say it's low-priority, so we can file an issue for it.

Reply:
I don't think there's anything wrong with using the Unicode π (I've seen it used in one of the files actually). Qiskit seems to use `:math:` for their docstrings and Cirq seems to just use LateX mode with `$$ $$` afaict. For me, I guess I wanted to stick to one standard rendering mode for all math symbols but most docstrings don't have many explicit math expressions/formulas to begin with so not really an issue to use Unicode characters once off.

Reply:
agree re: choosing something consistent. there also might be accessibility-related reasons to prefer one or another, which we should look into

i think all else being equal i generally prefer unicode characters, especially if it's just for e.g. a single greek letter (usually i'm reading docstrings in a terminal/text editor, which will render unicode `π` but not e.g. ``:math:`-\pi\`:``. but for this pr i think whatever seems most consistent with what we have is fine :)

Reply:
(otherwise this pr looks good to me, will approve)